pkg/authorization/authorizer/interfaces.go

```diff
@@ -10,19 +10,19 @@ import (
 )
 
 type Authorizer interface {
-	Authorize(ctx kapi.Context, a AuthorizationAttributes) (allowed bool, reason string, err error)
-	GetAllowedSubjects(ctx kapi.Context, attributes AuthorizationAttributes) (sets.String, sets.String, error)
+	Authorize(ctx kapi.Context, a Action) (allowed bool, reason string, err error)
+	GetAllowedSubjects(ctx kapi.Context, attributes Action) (sets.String, sets.String, error)
 }
 
 type AuthorizationAttributeBuilder interface {
-	GetAttributes(request *http.Request) (AuthorizationAttributes, error)
+	GetAttributes(request *http.Request) (Action, error)
```

Comment:
It is but the refactor here is incredibly invasive.  Changing it to Attributes doesn't help the scope of the refactor.  I think we should queue this up for post 1.3